.github/workflows/build_trtllm.yaml

```diff
@@ -0,0 +1,151 @@
+name: Build TensorRT-LLM
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        type: string
```

Comment:
Please make CI not read the runner in inputs. This is terribly unsafe (and not needed here).

You're directly reading this as the `runs-on` primitive. We're never doing that.